core-services/ci-secret-generator/_config.yaml

```diff
@@ -180,3 +180,13 @@
     - build02
     service_account:
     - sprint-automation
+- fields:
+  - cmd: '{ oc --kubeconfig /tmp/new.sa.$(service_account).$(cluster).config login $(oc --kubeconfig /tmp/sa.config-updater.$(cluster).config config view -o jsonpath="{.clusters[0].cluster.server}") --token $(oc --kubeconfig /tmp/sa.config-updater.$(cluster).config create token -n ci $(service_account) --duration=2419200s) > /dev/null; cat /tmp/new.sa.$(service_account).$(cluster).config; } | sed "s/$(service_account)/$(cluster)/g"'
```

Comment:
This smells: pipe, redirect, ... shell embeddedin yaml.

Maybe I should make a shell available in the image.